src/main/java/Chronos.java

```diff
@@ -0,0 +1,355 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import task.Task;
+import task.Todo;
+import task.Deadline;
+import task.Event;
+
+/**
+ * Represents the main class of the Chronos Task Management System.
+ */
+public class Chronos {
+    private static final String DIVIDER = "        ------------------------------------------------------------";
+    private static final String POSSIBLE_COMMANDS = "        TODO     --- todo [task name]\n" +
+                                                    "        DEADLINE --- deadline [task name] /by [yyyy-mm-dd HH:MM]\n" +
+                                                    "        EVENT    --- event [task name] /from [yyyy-mm-dd HH:MM] /to [yyyy-mm-dd HH:MM]" ;
+
+    private static String filePath = "./data/chronos.txt";
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    private static int noOfTasks = 0;
```

Comment:
Is this value necessary to use? As you may keep track of the noOfTasks by just calling tasks.size();